gtsfm/two_view_estimator.py

```diff
@@ -278,8 +255,10 @@ def aggregate_frontend_metrics(
     for report in two_view_reports_dict.values():
         if report is None:
             continue
-        rot3_angular_errors.append(report.R_error_deg)
-        trans_angular_errors.append(report.U_error_deg)
+        if report.R_error_deg is not None:
+            rot3_angular_errors.append(report.R_error_deg)
```

Comment:
@travisdriver we shouldn't need this `is not None` check here, since we are just filling the array with None, and then casting it to float which converts them to Nan, and then they are ignored in the `np.nanmean` call.

Reply:
But if keeps the types simpler, maybe it's ok.

Reply:
Was trying to fix some of the mypy errors

Reply:
Got it, sure.